Development_Documents/Issue_Management_Process.md

```diff
@@ -0,0 +1,48 @@
+# Issue Management Process
+The [Development Workflow](Development_Workflow) document describes the concept by which changes are proposed, reviewed, and prioritized. This document expands on that specifically by defining the standard practices used by BOINC community within the Github issue tracking system.
+
+## 1. Creating Issues
+Whether submitting a new issue as a bug report or a feature request, the submission should contain the same general information and will follow the same process. The information to include in any new issue would be:
+- a title
+- a description
+- appropriate labels to indicate the BOINC components that the issue affects (these labels are prefixed by "C: ", e.g. "C: Manager" for the BOINC manager desktop software, or "C: Server - Validator" for the validator component of the BOINC server)
+
+The assignee should only be set when someone has taken ownership of the issue, as issues without an assignee will show up in a list of issues waiting for someone to begin work on them.
+
+### 1.1 Issue Review
+Issues should be reviewed prior to development beginning on them. Other members on the BOINC community should look for issues that have not been reviewed and determine, if they are able, whether an issue is suitable for development now, in the future, or not at all. The following labels determine the review status:
+- Verified: a good candidate for current development
+- Discussion Needed: bring up on the BOINC Contributors call for discussion
+- Parked: likely to be a candidate for development in the future, but not now
```

Comment:
I think that _parked_ mixes status and prioritization. We have the labels P: and E: for prioritization and estimation respectively. This should be mentioned here in the document. Whether an issue is being working on is more to do with someone's willingness to get it done rather than someone else's judgement on the issue. _Discussion_ is really _Needs Consensus_, discussion is just part of that, a vote could be another approach. I recently started using the label _validate_ for issues that need testing.